mapie/metrics.py

```diff
@@ -307,3 +308,397 @@ def top_label_ece(
         )
     ece /= len(labels)
     return ece
+
+
+def regression_coverage_score_v2(
+    y_true: NDArray,
+    y_intervals: NDArray,
+) -> NDArray:
+    """
+    Effective coverage score obtained by the prediction intervals.
+
+    The effective coverage is obtained by estimating the fraction
+    of true labels that lie within the prediction intervals.
```

Comment:
Add a brief comment to explain why it is different from the previous version (regression_coverage_score_v2 uses directly the output of predict method = intervals)

Reply:
Why is it resolved?